tensorflow_encrypted/protocol/pond.py

```diff
@@ -2371,6 +2399,57 @@ def _stack_masked(prot, xs: List[PondMaskedTensor], axis: int=0):
     )
 
 
+def _concat_shared(prot, xc: Union[List[PondPublicTensor], List[PondPrivateTensor]], axis):
```

Comment:
Is it possible to use the PondTensor annotation here?

Also, it is not completely annotated is it on purpose?

Reply:
Can't use PondTensor here I don't think. It doesn't accept a masked tensor.